contracts/Arbitrum_SpokePool.sol

```diff
@@ -0,0 +1,118 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./SpokePool.sol";
+import "./SpokePoolInterface.sol";
+
+interface StandardBridgeLike {
+    function outboundTransfer(
+        address _l1Token,
+        address _to,
+        uint256 _amount,
+        bytes calldata _data
+    ) external payable returns (bytes memory);
+}
+
+/**
+ * @notice AVM specific SpokePool.
+ * @dev Uses AVM cross-domain-enabled logic for access control.
+ */
+
+contract Arbitrum_SpokePool is SpokePoolInterface, SpokePool {
+    // Address of the Arbitrum L2 token gateway.
+    address public l2GatewayRouter;
+
+    // Admin controlled mapping of arbitrum tokens to L1 counterpart. L1 counterpart addresses
+    // are neccessary to bridge tokens to L1.
+    mapping(address => address) public whitelistedTokens;
```

Comment:
pain that we need this just for arbitrum but works kinda well given we have separate contracts for each chain.